utils/measure/measure.py

```diff
@@ -912,23 +920,48 @@ def create(self) -> PowerMeter:
 def main():
     print(f"Powercalc measure: {_VERSION}\n")
 
+    device = inquirer.list_input("What kind of device do you want to measure the power of?",
+                              choices=[cls.value for cls in DeviceType])
+
     light_controller_factory = LightControllerFactory()
     power_meter_factory = PowerMeterFactory()
 
     try:
         power_meter = power_meter_factory.create()
         light_controller = light_controller_factory.create()
-    
+
         measure = Measure(light_controller, power_meter)
 
         args = sys.argv[1:]
-        if not args:
+
+        if device == DeviceType.LIGHT:
             measure.start()
             exit(0)
-
-        if args[0] == "average":
+        elif device == DeviceType.SPEAKER:
+            summary = {}
+            try:
+                if args[0] == "average":
+                    duration = int(args[1])
+            except IndexError:
+                duration = 20
+            if inquirer.confirm('Ready to measure the standby-power? (Make sure your devices is in off or idle state in HA)'):
```

Comment:
```suggestion
            if inquirer.confirm('Ready to measure the standby-power? (Make sure your devices is in off or idle state in HA)', default=True):
```

Set default to Y so the user only has to hit enter.